homeassistant/components/venstar/sensor.py

```diff
@@ -58,6 +63,11 @@
     RUNTIME_OV: "Override",
 }
 
+CONSUMABLE_ATTRIBUTES = {
+    CONSUMABLE_FILTER_HOURS: {"name": "Filter Used Hours", "uom": UnitOfTime.HOURS},
+    CONSUMABLE_FILTER_DAYS: {"name": "Filter Used Days", "uom": UnitOfTime.DAYS},
+}
+
```

Comment:
Now I'm reading the PR description again I'm assuming the days means "days this has been installed" and hours means "hours of active use". I think we should make this clearer in the names. Preferably the names should not contain the unit of measurement, because like said, that is changeable by the user and looks strange to suddenly measure "Filter Used Days" in weeks.

Reply:
I was mimicing how the existing "runtime" devices/attributes are.

Reply:
How do you mean runtime?